.gitignore

```diff
@@ -19,6 +20,10 @@ py27_gevent/
 py27_native/
 py3[0-9]_gevent/
 py3[0-9]_native/
+src/python/grpcio_channelz/LICENSE
+src/python/grpcio_health_checking/LICENSE
+src/python/grpcio_reflection/LICENSE
+src/python/grpcio_status/LICENSE
```

Comment:
Discussed offline with @lidizheng . These lines were added to the `.gitignore` because building distributions for each of these packages requires the license to be present in the source directory for the package.

However, since we only want one `LICENSE` file to stand as the source of truth for the repo, we are currently copying them from the `grpcio` package to every other package directory. This leaves some files not checked into the source repo.

We discussed an alternative solution. Instead of copying, we will check in symlinks to each package besides grpcio. I will implement this in a forthcoming PR.

Reply:
PR up: https://github.com/grpc/grpc/pull/17752